docs/distributor-notes.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+nav_order: 8
```

Comment:
Nit: I'd probably put this below Supported Platforms since that's a user-facing doc and this isn't.